src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -141,17 +164,24 @@
   "RootFolderMissingHealthCheckMessage": "Missing root folder: {0}",
   "RootFolderMultipleMissingHealthCheckMessage": "Multiple root folders are missing: {0}",
   "SearchForMonitoredEpisodes": "Search for monitored episodes",
+  "Series": "Series",
   "SetTags": "Set Tags",
+  "Settings": "Settings",
   "ShowAdvanced": "Show Advanced",
   "ShownClickToHide": "Shown, click to hide",
   "SizeOnDisk": "Size on disk",
+  "System": "System",
   "SystemTimeHealthCheckMessage": "System time is off by more than 1 day. Scheduled tasks may not run correctly until the time is corrected",
   "Tags": "Tags",
+  "Tasks": "Tasks",
+  "UI": "UI",
   "UI Language": "UI Language",
   "Unmonitored": "Unmonitored",
+  "Updates":  "Updates",
   "UpdateAvailableHealthCheckMessage": "New update is available",
   "UpdateStartupNotWritableHealthCheckMessage": "Cannot install update because startup folder '{0}' is not writable by the user '{1}'.",
   "UpdateStartupTranslocationHealthCheckMessage": "Cannot install update because startup folder '{0}' is in an App Translocation folder.",
   "UpdateUINotWritableHealthCheckMessage": "Cannot install update because UI folder '{0}' is not writable by the user '{1}'.",
+  "Wanted": "Wanted"
```

Comment:
I think the missing comma at the end is causing the test failures.

Reply:
> I think the missing comma at the end is causing the test failures.

oops :D